cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/mod.rs

```diff
@@ -14,6 +14,7 @@
 // limitations under the License.
 
 mod aliases;
+mod bridges;
```

Comment:
very nit, I used `bridging.rs` for bridge hub tests :D

```suggestion
mod bridging;
```